### PR TITLE
Added a new todo activity

### LINKED ISSUE #1: Create a New ToDo Activity
Create a new activity to take input from user for a new task.
The fields are : -
1. Task Name (EditText)
2. Description (EditTExt)
3. Deadline (For now, an edit text string in form dd/mm/yyyy)
4. Enable Notification (Just a radio button)
5. Priority (An integer value from 1-5)
and Submit button.
Just make the layout of the activity with these fields, don't worry about database and intents for now.

### PR DESCRIPTION
**Pull request by chanmol1999**
fixes #1 
Before submitting a pull request, please make sure the following is done:

	•	Fork the repository and create your branch from master.

	•	Include a human-readable description of what the pull request is trying to accomplish.

	•	If you've added code that should be tested, add tests and make sure the test suite passes.

	•	Steps for the reviewer(s) on how they can manually QA the changes.

	•	Have a fixes #XX reference to the issue that this pull request fixes.

### CODE REVIEW

File: app/src/main/res/menu/add.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <item
+        android:id="@+id/add"
+        android:title="Submit"
+        app:showAsAction="ifRoom"></item>
```
Comment (shiv07tiwari) [inline app/src/main/res/menu/add.xml]: Change this ifRoom to always.

**Comment by shiv07tiwari** [comment]
@chanmol1999 Make the change.
Also, tell me something : 
1. Are you registered on OpenCode Participants List ? (Google form)
2. Did you add yourself to participants list ? (Issue in Collab Website)
3. Are you on to-do slack channel ?

**Comment by shiv07tiwari** [comment]
@chanmol1999  Join Zulip channel of OpenCode.
https://opencode.zulipchat.com/ 
It is compulsory.
Tell me when you are done, I'll merge the PR

**Comment by chanmol1999** [comment]
Done

**Comment by shiv07tiwari** [comment]
Good job @chanmol1999 .

**Pull request merged**